src/main/scala/io/chrisdavenport/mules/Cache.scala

```diff
@@ -98,12 +101,39 @@ object Cache {
     def unsafeFromNanos(l: Long): TimeSpec =
       new TimeSpec(l)
 
+    implicit class TimeSpecOps(val timeSpec: TimeSpec) {
```

Comment:
Why is this an implicit class?

Reply:
Just to use it as an extension method (syntax). I can change it to a common method if you prefer.